submodules/command_ingest/__init__.py

```diff
@@ -1,114 +1,102 @@
 import logging
+from inspect import signature
+from collections import deque as queue
+
+# Registered commands
+# import core
+# from submodules import aprs, iridium, telemetry
 
 logger = logging.getLogger("CI")
 
 total_received: int = 0
 total_errors: int = 0
 total_success: int = 0
 
-# All currently registered commands.  The key is the function identifier in the air, and values follow
-# the format {"function": func, "args": args}  where args is an array of types (e.g. int, str, float, etc.)
-registered_commands = {}
 
+general_queue = queue()
 
 # Annotation to register a function as a command
 # Example usage:
 # @command("aprs_set_bperiod", int)
 # def set_bperiod(period: int) -> str:
 # ...
-
-def command(command_name, *args, **kwargs):
+def associate(cmd: str):
     """
     Given a function name, return a decorator that registers that function.
     :return: Decorator that registers function.
     """
-    def decorator(func):
-        registered_commands[command_name] = {"function": func, "args": args}
-        return func
 
-    return decorator
+    valid_commands = {
```

Comment:
This is good for now, but make sure there is functionality for adding actual pFS commands in a later change